mobile/components/views/Attractions/Router.tsx

```diff
@@ -1,31 +1,45 @@
-import { useState } from "react";
-import {DrawerItem} from '@react-navigation/drawer';
-import { View } from "react-native";
+import React, { useState } from 'react';
+import { DrawerItem, DrawerNavigationProp } from '@react-navigation/drawer';
+import { View } from 'react-native';
 
+type DrawerParamList = {
+  Search: any;
+  AddNew: undefined;
+};
 
+type AttractionSubMenuProps = {
+  navigation: DrawerNavigationProp<DrawerParamList>;
+};
 
-const AttractionSubMenu: React.FC<any> = (props) => {
-    const [nestedMenuOpen, setNestedMenuOpen] = useState(false);
+const AttractionSubMenu: React.FC<AttractionSubMenuProps> = (props) => {
+  const [nestedMenuOpen, setNestedMenuOpen] = useState(false);
```

Comment:
maybe it woul be better to call it something like nestedMenuToggle, Toggle means that you can open or close something.

Reply:
done